cfgov/cfgov/settings/__init__.py

```diff
@@ -0,0 +1 @@
+from .local import *
```

Comment:
Yes that's true but I would think that we could change it in the deployment somehow. The problem with doing it the other way is that you can't import any settings from anywhere because you can't be sure which settings are being used.